src/Microsoft.DotNet.Interactive/Kernel.Static.cs

```diff
@@ -73,7 +73,7 @@ private static async Task<string> GetInputAsync(string prompt, bool isPassword,
     public static void CSS(string content) =>
         // From https://stackoverflow.com/questions/524696/how-to-create-a-style-tag-with-javascript
         Javascript($@"
-            var css = '{content}',
+            var css = `{content}`,

```

Comment:
I was considering removing this method altogether, along with the `Javascript` method. People should likely be emitting `HTML` elements for this or calling a JavaScript kernel if one is available.

Reply:
this is blocking some notebooks development as that is public api. I'd fix it and mark as deprecated?

Reply:
but I think is very convenient shortcut to be able to sue the `CSS` helper to add styles

Reply:
Convenient isn't sufficient if it doesn't work consistently.

Reply:
I'd suggest we leave it for now, with the fix, but should discuss whether this is the right design, especially given the Pythonesque top-level function name.